src/averageAges.js

```diff
@@ -37,7 +47,17 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const listToCalculate = withChildren
+    ? people.filter(
+      person => person.sex === 'f' && people.some(
+        child => child.mother === person.name))
+    : people.filter(person => person.sex === 'f', 0);
```

Comment:
Here you will be able to use your helper function ;)

Reply:
because of human rights i cant divide people by sex before ) Besides, I coded separate functions as they asked in task (with strong wish to make "Helper"). 
and if it will be super needed i will redo.